content/03.categorize.md

```diff
@@ -5,18 +5,8 @@ Once diagnosed with a disease, an individual might be assigned a stage based on
 While these rules are refined over time, the process is evolutionary and ad hoc, potentially impeding the identification of underlying biological mechanisms and their corresponding treatment interventions.
 
 Deep learning methods coupled with a large corpus of patient phenotypes may provide a more data-driven approach to patient categorization.
-A deep neural network has the potential to identify entirely new categories of health or disease that are only present when data from multiple lab tests are integrated.
-
-As an example, consider the condition Latent Autoimmune Diabetes in Adults
-(LADA; reviewed in [@doi:10.2337/diabetes.54.suppl_2.S68]).
-In the absence of a pre-specified disease definition, a deep neural network might have identified a subgroup of individuals with blood glucose levels that indicated diabetes as well as auto-antibodies, even though the individuals had never been diagnosed with type 1 diabetes -- the autoimmune form of the disease that arises in young people.
-Such a neural network would be identifying patients with LADA.
-As no such computational approach existed, LADA was actually identified by Groop et al. [@doi:10.2337/diab.35.2.237].
-
-One should not regard recapitulation of existing disease categories as a gold-standard for deep learning results.
-Instead, a meaningful contribution to patient categorization would be to identify new shared mechanisms that would otherwise be obscured due to ad hoc historical definitions of disease.
+A meaningful contribution to patient categorization would be to identify new shared mechanisms that would otherwise be obscured due to ad hoc historical definitions of disease.
 Perhaps deep neural networks, by reevaluating data without the context of our assumptions, can reveal novel classes of treatable conditions.
-
```

Comment:
I suggest keeping this newline so that `In spite of such optimism` starts a new paragraph.

Reply:
added it back 👍